### PR TITLE
docs(blog): 1.15 release

### PR DESCRIPTION
**Pull request by ikatyang**
There're 50+ items in this release, it'd be appreciated if somebody could pick some of them up.

Feel free to edit this PR directly.

### 🌟 [Live preview](https://deploy-preview-5296--prettier.netlify.com/blog/2018/11/07/1.15.0.html) 🌟

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@
   · SCSS
   · Less
   · JSX
+  · HTML
   · Vue
+  · Angular
   · GraphQL
   · JSON
   · Markdown
```
Comment (j-f1) [inline README.md]: This line is starting to get a little long. Should it be reformatted somehow?

File: docs/options.md [hunk 2]
```diff
@@ -263,3 +262,19 @@ Valid options:
 | Default      | CLI Override                                                | API Override                                                |
 | ------------ | ----------------------------------------------------------- | ----------------------------------------------------------- |
 | `"preserve"` | <code>--prose-wrap <always&#124;never&#124;preserve></code> | <code>proseWrap: "<always&#124;never&#124;preserve>"</code> |
+
+## HTML Whitespace Sensitivity
+
+_available in v1.15.0+_
```
Comment (j-f1) [inline docs/options.md]: I wonder if these should be formatted differently? Perhaps:

> *First available in vX.X.X. Deprecated in vY.Y.Y*

**Comment by ikatyang** [comment]
You can edit this PR directly, I've checked <kbd>Allow edits from maintainers.</kbd>.

File: docs/options.md (hunk 2 above)
Comment (ikatyang) [reply docs/options.md]: Since there's a `+`, I guess it should be fine?

**Comment by lipis** [comment]
Thanks @ikatyang.. I was experimenting.. :)

File: README.md (hunk 1 above)
Comment (ikatyang) [reply README.md]: IMO, JavaScript should be the first one, sorting by group should be better.

Comment (lipis) [reply README.md]: We can have **JavaScript** in bold :)

Comment (lipis) [reply README.md]: WDYT? https://github.com/prettier/prettier/blob/b944ad58eed30d0dc7abc347e744dbfbe919891d/README.md#opinionated-code-formatter

**Comment by lipis** [comment]
Maybe grouping the languages? https://github.com/prettier/prettier/blob/b944ad58eed30d0dc7abc347e744dbfbe919891d/README.md#opinionated-code-formatter

**Comment by alexw10** [comment]
I think I found a bug in 1.15 in LESS.

➜  ~/projects/ClinicianSite prettier -v
1.15.0-dev
Less file is this:

@import 'theme-variables';

// body

body {
  background-color: @body-background-color;
  font-family: @base-font-family;
  color: @body-text-color;
}
Run --write on less file:

➜  ~/projects/ClinicianSite prettier --write /Users/wintea7/projects/ClinicianSite/src/styles/theme.less
src/styles/theme.less 169ms
Should now pass the -l option but does not:

➜  ~/projects/ClinicianSite prettier -l /Users/wintea7/projects/ClinicianSite/src/styles/theme.less
src/styles/theme.less
Also, something I observed is everytime I run the --write it adds a new line under the comment // body Also, if I remove the comment // body it then passes. So it seems to have something to do with comments in less files that it is not handling this properly right now.

If there is a better PR to put this on please let me know.

**Comment by j-f1** [comment]
@alexw10 Can you open an issue please?

**Comment by alexw10** [comment]
> @alexw10 Can you open an issue please?

Will do should I remove my comment above?

**Comment by j-f1** [comment]
> should I remove my comment above?

No need to. I can collapse them.

**Comment by ikatyang** [comment]
I originally thought HTML shouldn't be that hard, we could finish it quickly. 😭

**Comment by lipis** [comment]
Just merged this one if we want to add it: https://github.com/prettier/prettier/pull/5250

File: website/blog/2018-10-00-1.15.0.md [hunk 3]
```diff
@@ -0,0 +1,1766 @@
...
+// Output (Prettier 1.15)
+const data = { foo: 'bar', baz: 'quux' }
+
+```
+
+### JSX
+
+#### fix: splitting jsx text correctly ([#5006] by [@yuliaHope])
+
+TODO
```
Comment (ikatyang) [inline website/blog/2018-10-00-1.15.0.md]: @lydell Can you help me with this item? as I don't have too much context on it.

Comment (lydell) [reply website/blog/2018-10-00-1.15.0.md]: Ok!

Comment (lydell) [reply website/blog/2018-10-00-1.15.0.md]: I've pushed a description. I hope I got it right, I'm in a bit of a hurry

Comment (ikatyang) [reply website/blog/2018-10-00-1.15.0.md]: 👍 Thanks!

**Comment by ikatyang** [comment]
TODO items are done, ready for review.

**Comment by ikatyang** [comment]
Sure. In the last release, I merged the blog post before the release, which results in no new version available on npm for 12 min. 😂

**Comment by lipis** [comment]
It looks like we will have to add this one as well: #5327 :)

**Comment by lydell** [comment]
@lipis It might be better to hold off on that PR to the next release.

**Comment by lipis** [comment]
@lydell Just comment on the original issue.. we didn't add the 1.15 milestone to it anyway.

**Pull request merged**